test/units.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+go get github.com/weaveworks/weave/testing/cover
```

Comment:
I'll add a build target for it then.

On Sat, Jul 11, 2015 at 11:14 AM, Matthias Radestock <
notifications@github.com> wrote:

> In test/units.sh
> <https://github.com/weaveworks/weave/pull/1115#discussion_r34411515>:
>
> > @@ -0,0 +1,28 @@
> > +#!/bin/bash
> > +
> > +go get github.com/weaveworks/weave/testing/cover
>
>  My understanding is that go get will use the local code
>
> Yes, that is my understanding too. But it still looks weird.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/weave/pull/1115/files#r34411515>.
>